### PR TITLE
Bugfix/add case insensitive extension check

### PR DESCRIPTION
**Pull request by alduinsh**
please check two:
1. api_utils  - >  method '__allowed_file'
2. main_test - >  test 'test_allowed_file'

### CODE REVIEW

File: main_test.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 from fastapi.testclient import TestClient
 from main import app
 from utils.api_constants import DOWNLOADS_FOLDER, RESULT_FILE_NAME, VIDEOS_FOLDER, IMAGES_FOLDER
-from utils.enums import HashAlgorithm
+from utils.enums import HashAlgorithm, SourceType
 from utils.hasher import Hasher
-
+from utils.api_utils import allowed_file
```
Comment (Illania) [inline main_test.py:15]: Some import were missing, I added them. You should test before pushing.

File: main_test.py [hunk 2]
```diff
@@ -84,3 +84,18 @@ def test_swap_multi():
     # Hasher.check_hash_equals(HashAlgorithm.SHA1, result_file_path, '../demo/multi/output.mp4')
     if os.path.exists(result_file_path):
         os.remove(result_file_path)
+
+def test_allowed_file():
+   """Test allowed file types"""
+   assert allowed_file("test.jpg", SourceType.IMAGE) == True
```
Comment (Illania) [inline main_test.py:90]: It is enough to test only allowed_file function, no need to test system functions like lower(), they certainly work as intended, so I removed unneeded checks

**Review by Illania — APPROVED** [review]
I fixed the small issues, now it works well, good job!

**Review by Illania — APPROVED** [review]
works well!

**Review by gurudevapp — APPROVED** [review]

**Pull request merged**